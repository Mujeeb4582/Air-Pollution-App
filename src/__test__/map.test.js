import React from 'react';
import { render } from '@testing-library/react';
import MapChart from '../Components/map';

test('renders map chart component', () => {
  const { container } = render(<MapChart />);
  expect(container).toMatchSnapshot();
});
