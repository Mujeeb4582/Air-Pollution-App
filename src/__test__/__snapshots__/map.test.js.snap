// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders map chart component 1`] = `
<div>
  <svg
    class="rsm-svg "
    style="fill: #2a354d; background-color: rgb(218, 76, 122);"
    viewBox="0 0 800 600"
  >
    <g
      class="rsm-geographies "
      data-testid="map"
    />
  </svg>
</div>
`;
